src/domain/service/note.ts

```diff
@@ -35,6 +35,15 @@ export default class NoteService {
     });
   }
 
+  /**
+   * Deletes note by id
+   * @param id - note internal id
+   * @returns { Promise<boolean | null> }
+   */
+  public async deleteNoteById(id: NoteInternalId){
```

Comment:
specify return type in TS instead of jsdoc

Reply:
fixed